Assets/Scripts/Models/World.cs

```diff
@@ -63,7 +64,7 @@ public World(int width, int height)
         SetupWorld(width, height);
         int seed = UnityEngine.Random.Range(0, int.MaxValue);
         WorldGenerator.Generate(this, seed);
-
+		Debug.Log ("Generated World");
```

Comment:
It looks like your editor is still configured to use tabs as tabs, instead of tabs as spaces.  Please check CONTRIBUTING.md for the preferred settings for this project.